tests/test_schema.py

```diff
@@ -1534,20 +1534,6 @@ def schema_extra(schema):
     assert Model.schema() == {'title': 'Model', 'type': 'override'}
 
 
-def test_model_with_schema_extra_callable_classmethod_asserts():
```

Comment:
let's keep this test but demonstrate that it works.

Reply:
Done.